src/content/quickstarts/chainlink-demo-app.mdx

```diff
@@ -0,0 +1,138 @@
+  production environment without completing your own audits and application of best practices. Neither Chainlink Labs,
+  the Chainlink Foundation, nor Chainlink node operators are responsible for unintended outputs that are generated due
+  to errors in code.
+</Aside>
+
+## Objective
+
+This guide shows you how to deploy a demo app, which requires several steps common to Web3 apps. You will learn the basics for how to use the Hardhat development environment and the Next.js frontend framework.
+
+## Before you begin
+
+Before you start this tutorial, complete the following items:
+
+- If you are new to smart contract development, learn how to [Deploy Your First Smart Contract](/getting-started/deploy-your-first-contract).
+- Set up a cryptocurrency wallet such as [MetaMask](https://metamask.io/).
+- To deploy this contract on testnets, ensure the deployer account has testnet ERC-677 LINK. Use the [LINK faucet](https://faucets.chain.link/) to retrieve testnet LINK.
+- [Install Node](https://nodejs.org/en/download/)
+- [Install Yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
+- [Install Git](https://git-scm.com/downloads)
+
+## Steps to implement
+
+### Configure your local files
+
+1. Clone the repo and change directories:
+
+   ```bash
+   git clone https://github.com/smartcontractkit/chainlink-fullstack && cd chainlink-fullstack
+   ```
+
+1. Init the submodule:
+
+   ```bash
+   git submodule init
+   ```
+
+1. Update the submodule:
+
+   ```bash
+   git submodule update
+   ```
+
+1. Install the dependencies:
+
+   ```bash
+   yarn install
+   ```
+
+1. Start up the local Hardhat network and deploy all contracts:
+
+   ```bash
+   yarn chain
+   ```
+
+1. In a second terminal start up the local development server run the front-end app:
+
+   ```bash
+   yarn dev
+   ```
+
+To interact with the local network, follow this step-by-step guide on how to use [MetaMask with a Hardhat node](https://support.chainstack.com/hc/en-us/articles/4408642503449-Using-MetaMask-with-a-Hardhat-node).
+
+If you've set the mnemonic from MetaMask the first 20 accounts will be funded with ETH.
+
+### Configure environment variables
+
+1. Use the `.env.example` file in the `hardhat` workspace. Copy it to new `.env` files and replace the values with the following values:
+
+- `NETWORK_RPC_URL` - An RPC is required to deploy to public networks. You can obtain one from [Infura](https://infura.io).
+- `MNEMONIC` - This is used to derive accounts from a wallet seed phrase from Metamask. The first account must have enough ETH to deploy the contracts as well as LINK which can be obtained from [Chainlink's faucets](https://faucets.chain.link).
+- `PRIVATE_KEY` - This is an alternative to using the mnemonic. Some changes are required in `hardhat.config.js`
+- `ETHERSCAN_API_KEY` - This can be obtained from an Etherscan account, and is used to verify your contract code on Etherscan.
```

Comment:
```suggestion
    - `NETWORK_RPC_URL` - An RPC is required to deploy to public networks. You can obtain one from [Infura](https://infura.io).
    - `MNEMONIC` - This is used to derive accounts from a wallet seed phrase from Metamask. The first account must have enough ETH to deploy the contracts as well as LINK which can be obtained from [Chainlink's faucets](https://faucets.chain.link).
    - `PRIVATE_KEY` - This is an alternative to using the mnemonic. Some changes are required in `hardhat.config.js`
    - `ETHERSCAN_API_KEY` - This can be obtained from an Etherscan account, and is used to verify your contract code on Etherscan.
```

Reply:
Indentation so it doesn't break the numbered list